app/models/role.rb

```diff
@@ -9,5 +9,13 @@ class Role < ActiveRecord::Base
           class_name: 'RoleDistance',
           foreign_key: 'contract_role_distance_id'
 
+  belongs_to :permanent_role_travel_willingness_option, 
+          class_name: 'RoleTravelWillingnessOption',
+          foreign_key: 'contract_role_travel_willingness_option_id'
+
+  belongs_to :contract_role_travel_willingness_option, 
+          class_name: 'RoleTravelWillingnessOption',
+          foreign_key: 'permanent_role_travel_willingness_option_id'
```

Comment:
wait, are these foreign_keys the wrong way around? @radicalise, or am I going crazy